js/lib/beautify-css.js

```diff
@@ -234,6 +241,16 @@
             return false;
         }
 
+        function removeWhiteSpaceOnEmptyLines(input) {
```

Comment:
I don't think this actually has any effect the way the newlines are handled now. It's highly likely this is an expensive `noop`. I'll take a look tonight.